src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -97,7 +108,7 @@ public async Task<IActionResult> Authorize()
                     {
                         [OpenIddictServerAspNetCoreConstants.Properties.Error] = Errors.ConsentRequired,
                         [OpenIddictServerAspNetCoreConstants.Properties.ErrorDescription] =
-                            "The logged in user is not allowed to access this client application."
+                            S["The logged in user is not allowed to access this client application."]
```

Comment:
⚠️ 

https://github.com/OrchardCMS/OrchardCore/blob/19f34caa1bf9b6c633146da80e340ff119424e15/src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs#L30-L31

Reply:
Thanks. I undid this.